src/Symfony/Component/Config/Definition/PrototypedArrayNode.php

```diff
@@ -168,7 +168,7 @@ public function addChild(NodeInterface $node)
     protected function finalizeValue($value)
     {
         if (false === $value) {
-            throw new UnsetKeyException(sprintf('Unsetting key for path "%s", value: %s.', $this->getPath(), json_encode($value)));
+            throw new UnsetKeyException(sprintf('Unsetting key for path "%s", value: "%s".', $this->getPath(), json_encode($value)));
```

Comment:
Not sure about this one, I would not put quotes around a boolean.
If it was reported by fabbot, please revert. Sometimes he is wrong for such cases

Reply:
done

Reply:
thanks